doc/gov/chapters/chapter04.md

```diff
@@ -4,66 +4,106 @@
+### 4.2.5 Interim Leads/Co-Leads
+
+- In case a Lead/Co-Lead steps down, an interim person (proposed by the person who is stepping down) to take over position duties until next available election.
+- For new Work Streams, a new Lead/Co-Leads will be assigned by Technical Steering/Governance to be interim until next available election period.
+
 <a name="4.3"></a>
 ## 4.3 Technical Steering Committee Lead/Co-Lead Nomination and Selection
 
-<a name="4.3.1"></a>
-### 4.3.1 Length of Terms
+The same terms, process of nomination and selection described in [**4.2**](#4.2) must be applied to technical steering committee leads and co-leads positions.
 
-<a name="4.3.2"></a>
-### 4.3.2 Representation
-
-<a name="4.3.3"></a>
-### 4.3.3 membership and voting rights
+Additionally, The following points also apply:
+- One Lead and up to 5 Co-Leads positions are available for the TSC leadership.
+- When the nomination and selection process is due within CNTT, all Technical Workstream nomination and selection process has to conclude **before** technical steering committee nomination and selection process starts.
+- Nomination and selection process for TSC should be facilitated by the Technical steering committee existing leads.
 
 <a name="4.4"></a>
 ## 4.4 Governance Committee Lead/Co-Lead Nomination and Selection
 
-<a name="4.4.1"></a>
-### 4.4.1 Length of Terms
+The same terms, process of nomination and selection described in [**4.2**](#4.2) must be applied to Governance Committee leads and co-leads positions.
 
-<a name="4.4.2"></a>
-### 4.4.2 Representation
+Additionally, the following points also apply:
+- One Lead and up to 3 Co-Leads positions are available for the Governance Comitee leadership.
+- When the nomination and selection process is due within CNTT, All Technical Workstream, Governance workstreams, and TSC nomination and selection process need to have been concluded **before** Governance committee nomination and selection process starts.
+- nomination and selection process for Governance Committee should be facilitated by the sitting Governance Committee Leads.
 
-<a name="4.4.3"></a>
-### 4.4.3 Membership and voting rights
 
 <a name="4.5"></a>
-## 4.5 Committee Structure
+## 4.5 Membership and voting rights
+
+Please refer to [Voting Representatives](../chapters/chapter05.md#52-voting-representatives)
 
 <a name="4.6"></a>
-## 4.6 Limits and Caps
+## 4.6 Committee Structure
 
-<a name="4.7"></a>
-## 4.7 Tracking contributions
+Please refer to [Org Structure](../chapters/chapter03.md#32-org-structure).
 
+<a name="4.7"></a>
+## 4.7 Limits and Caps
+- No Limit or CAPs for how many position a given individual or an organisation can hold.
+
+<a name="4.8"></a>
+## 4.8 Tracking contributions
+There are few scenarios where contributions needs to be tracked (e.g. to maintain voting rights for individuals or organisations). CNTT will actively look at the following contribution metrics and leverage any available automated tooling for such purposes:
```

Comment:
Typo: "... where contributions needs to ..." ---- "... where contributions need to ..."

Reply:
Fixed